### PR TITLE
Soarian r4 AllergyIntolerance Resource Documentation

### PR DESCRIPTION
**Pull request by VamsiMure**
Description
Soarian FHIR R4 AllergyIntolerance resource

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
Allergy-3
Allergy-4
Allergy-5
Allergy-6
Allergy-7
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: lib/resources/r4/soarian_allergy_intolerance.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
+name: AllergyIntolerance
+field_name_base_url: http://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance
+fields:
+- name: resourceType
+  required: 'Yes'
+  type: string
+  description: The type of the FHIR resource.
```
Comment (HRNWriter) [inline lib/resources/r4/soarian_allergy_intolerance.yaml:7]: Two changes:
- Delete the second "the" so the sentence reads "The type of FHIR resource."
- Add a registered trademark symbol after "FHIR".

File: lib/resources/r4/soarian_allergy_intolerance.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
+name: AllergyIntolerance
+field_name_base_url: http://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance
```
Comment (slvsridhar) [inline lib/resources/r4/soarian_allergy_intolerance.yaml:2]: Can you change the url to https://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance

File: lib/resources/r4/soarian_allergy_intolerance.yaml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    description: The code for an allergy statement (a positive or negated statement). 
+                 This may be a code for a substance or pharmaceutical product that is 
+                 considered to be responsible for the adverse reaction risk (for 
+                 example, Latex), an allergy condition (for example, Latex allergy), or 
+                 a general or categorical negated statement (for example, No known 
+                 allergy, No known drug allergies).
+    terminology:
+    - display: RxNorm
+      system: http://www.nlm.nih.gov/research/umls/rxnorm
+      info_link: http://hl7.org/fhir/r4/rxnorm.html
```
Comment (slvsridhar) [inline lib/resources/r4/soarian_allergy_intolerance.yaml:53]: Can you change the url to https://hl7.org/fhir/r4/rxnorm.html?

File: lib/resources/r4/soarian_allergy_intolerance.yaml [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    - display: SNOMED CT
+      system: http://snomed.info/sct
+      info_link: https://hl7.org/fhir/r4/valueset-allergyintolerance-code.html
+    - display: Unique Ingredient Identifier (UNII)
+      system: http://fdasis.nlm.nih.gov
+      info_link: http://hl7.org/fhir/R4/unii.html
```
Comment (slvsridhar) [inline lib/resources/r4/soarian_allergy_intolerance.yaml:59]: Can you change the url to https://hl7.org/fhir/R4/unii.html?

File: lib/resources/r4/soarian_allergy_intolerance.yaml (hunk 2 above)
Comment (VamsiMure) [reply lib/resources/r4/soarian_allergy_intolerance.yaml]: Done

File: lib/resources/r4/soarian_allergy_intolerance.yaml (hunk 3 above)
Comment (VamsiMure) [reply lib/resources/r4/soarian_allergy_intolerance.yaml]: Done

File: content/soarian/r4/clinical/summary/allergy-intolerance.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+# AllergyIntolerance
+
+* TOC
+{:toc}
+
+## Overview
+
+
+The AllergyIntolerance resource provides the clinical assessment of a patient’s allergy when exposed to a specific substance or class of substance, including information about the adverse reaction. _Soarian Clinicals_<sup>®</sup> does not differentiate between an allergy and an intolerance. The term __allergy__ is used generically and may overlap with an __intolerance__. Substances include medications, foods, and environmental allergens (such as plants and animals).
```
Comment (HRNWriter) [inline content/soarian/r4/clinical/summary/allergy-intolerance.md:13]: Remove italics from Soarian Clinicals.

File: content/soarian/r4/clinical/summary/allergy-intolerance.md [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+
+Allergy negation statements for No Known Allergies (NKA), No Known Drug Allergies (NKDA), and No Known Food Allergies (NKFA) are reported using the code element. If a patient has not been asked about their allergies, these statements are conveyed by the absence of information (empty query response). Allergy negation statements for latex and contrast are not coded and are conveyed in the response with the AllergyIntolerance.code.text element as __No known latex allergy (situation)__ or __No known contrast allergy (situation)__.
+
+_Soarian Clinicals_<sup>®</sup> supports a read-only Application Programming Interface (API). This API accepts `GET` and `POST` based [search] and `GET` based [read] interactions. The response represents the most current information about the patient that is charted in _Soarian Clinicals_<sup>®</sup> at the time of the query.
```
Comment (HRNWriter) [inline content/soarian/r4/clinical/summary/allergy-intolerance.md:17]: Remove italics from both references to Soarian Clinicals.

File: content/soarian/r4/clinical/summary/allergy-intolerance.md (hunk 5 above)
Comment (HRNWriter) [reply content/soarian/r4/clinical/summary/allergy-intolerance.md:13]: In the Overview section, "Soarian Clinicals" still has italics. Please remove this markdown.

PRODOC-160857.

File: content/soarian/r4/clinical/summary/allergy-intolerance.md (hunk 6 above)
Comment (HRNWriter) [reply content/soarian/r4/clinical/summary/allergy-intolerance.md:17]: In the Overview section, both references "Soarian Clinicals" still have italics. Please remove this markdown.

PRODOC-160857.

**Review by HRNWriter — COMMENTED** [review]
Please see my earlier comments. None of the references to Soarian Clinicals should be in italic font.

**Review by ashu-bdvt — APPROVED** [review]

File: content/soarian/r4/clinical/summary/allergy-intolerance.md (hunk 6 above)
Comment (VamsiMure) [reply content/soarian/r4/clinical/summary/allergy-intolerance.md]: Done

File: content/soarian/r4/clinical/summary/allergy-intolerance.md (hunk 5 above)
Comment (VamsiMure) [reply content/soarian/r4/clinical/summary/allergy-intolerance.md]: Done

**Review by HRNWriter — APPROVED** [review]
+1

**Review by slvsridhar — APPROVED** [review]

**Pull request merged**